src/pages/RichTextEditor.tsx

```diff
@@ -0,0 +1,35 @@
+import 'react-draft-wysiwyg/dist/react-draft-wysiwyg.css';
+
+import dynamic from 'next/dynamic';
+import { ComponentProps } from 'react';
+
+const Editor = dynamic(
+  () => import('react-draft-wysiwyg').then(({ Editor }) => Editor),
+  { ssr: false },
+);
+
+function RichTextEditor(props: ComponentProps<typeof Editor>) {
+  return (
+    <div
+      style={{
+        background: 'white',
+        borderRadius: 10,
+        padding: 10,
+        width: '100%',
+      }}
+    >
+      <Editor
+        {...props}
+        toolbar={{
+          options: ['inline', 'list', 'history'],
+          inline: {
+            inDropdown: false,
+            options: ['bold', 'italic', 'underline'],
+          },
+        }}
```

Comment:
```suggestion
        toolbar={{
          options: ['inline', 'list', 'history'],
          inline: {
            inDropdown: false,
            options: ['bold', 'italic', 'underline'],
          },
        }}
        {...props}
```

So props from higher up can override the `toolbar` prop, unless you want to prevent this.